Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/storage/entity/TraceLocationCheckInEntity.kt

```diff
@@ -9,47 +9,18 @@ import org.joda.time.Instant
 data class TraceLocationCheckInEntity(
     @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "id") val id: Long = 0L,
     @ColumnInfo(name = "guid") val guid: String,
-    @ColumnInfo(name = "guidHash") val guidHash: ByteArray,
+    @ColumnInfo(name = "guidHash") val guidHashBase64: String,
     @ColumnInfo(name = "version") val version: Int,
     @ColumnInfo(name = "type") val type: Int,
     @ColumnInfo(name = "description") val description: String,
     @ColumnInfo(name = "address") val address: String,
     @ColumnInfo(name = "traceLocationStart") val traceLocationStart: Instant?,
     @ColumnInfo(name = "traceLocationEnd") val traceLocationEnd: Instant?,
     @ColumnInfo(name = "defaultCheckInLengthInMinutes") val defaultCheckInLengthInMinutes: Int?,
-    @ColumnInfo(name = "traceLocationBytes") val traceLocationBytes: ByteArray,
+    @ColumnInfo(name = "traceLocationBytes") val traceLocationBytesBase64: String,
```

Comment:
is this working with the database column? it is still defined as `BOLB` there

Reply:
man, you are fast

Reply:
pushed the new json now